.devcontainer/devcontainer.json

```
@@ -9,7 +9,8 @@
       "version": "3.11"
     },
     "ghcr.io/devcontainers/features/docker-in-docker:2": {},
-    "./features/src/postgresql": {}
+    "./features/src/postgresql": {},
+    "ghcr.io/ministryofjustice/devcontainer-feature/aws:0": {}
```

```suggestion
    "ghcr.io/ministryofjustice/devcontainer-feature/aws:0": {},
    "ghcr.io/ministryofjustice/devcontainer-feature/static-analysis:0": {},
```
Adds static analysis, as per @Gary-H9's suggestion